### ISSUE TITLE
Cannot edit pages

### ISSUE THREAD

**Issue by elega9t** [Requarks/wiki#2410]
**Describe the bug**
Hi, I recently updated my wiki 2.4 installation to 2.5.126. I am able to create new pages, however editing existing pages fail with

```
Proxy Error

The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request GET /e/en/administration.

Reason: Error reading from remote server
```
I am using docker wiki installation.

**To Reproduce**
1. Create wiki instance with docker using wiki 2.4
2. Create a page
3. update wiki docker instance to 2.5
4. Try to edit the page
5. See error

**Expected behavior**
Editing existing pages should be possible.

**Host Info (please complete the following information):**
 - OS: Docker on Ubuntu 3.10.0-1127.19.1.el7.x86_64
 - Wiki.js version: 2.5.126
 - Database engine: mysql 8.0.21

**Comment by NGPixel** [comment]
Try without a reverse proxy.

**Comment by elega9t** [comment]
The edit request doesn't complete. Event after 7 min, the browser was still waiting for server response.

**Comment by Jamesits** [comment]
I'm having exactly the same issue -- all editor pages time out -- after upgrading to 2.5.126. My instance is reverse-proxied by an Apache and removing the reverse proxy is not feasible in my environment. 

System info:
* Wiki.js 2.5.126 (Official Docker image)
* NodeJS 14.9.0
* DB: MySQL 5.7.30-0ubuntu0.18.04.1

Already tried things that do not work:
* log out and re-login
* rebuild page tree
* re-render all pages
* flush pages and asset cache

**Comment by NGPixel** [comment]
Fixed by 5295e413be9d661e39b020521ff025f17933a13f. An update will be published shortly.
